builtin/providers/azurerm/resource_arm_loadbalancer.go

```diff
@@ -172,7 +177,18 @@ func resourecArmLoadBalancerRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("resource_group_name", id.ResourceGroup)
 
 	if loadBalancer.LoadBalancerPropertiesFormat != nil && loadBalancer.LoadBalancerPropertiesFormat.FrontendIPConfigurations != nil {
-		d.Set("frontend_ip_configuration", flattenLoadBalancerFrontendIpConfiguration(loadBalancer.LoadBalancerPropertiesFormat.FrontendIPConfigurations))
+		ipconfig := flattenLoadBalancerFrontendIpConfiguration(loadBalancer.LoadBalancerPropertiesFormat.FrontendIPConfigurations)
+		d.Set("frontend_ip_configuration", ipconfig)
+
+		for _, config := range ipconfig {
+			cfg := config.(map[string]interface{})
+			if priv_ip, ok := cfg["private_ip_address"]; ok {
```

Comment:
I think it'd be better to iterate over the `loadBalancer.LoadBalancerPropertiesFormat.FrontendIPConfigurations` here, rather than the flattened list?